inugami_webapp/src/main/webapp/css/core/component-jenkins.css

```
@@ -0,0 +1,215 @@
+    display: none;
+  }
+  .jenkins-jobs .column-list .job-state .item-name{
+    -webkit-transition: color  0.5s;
+    transition: color   0.5s;
+  }
+  .jenkins-jobs .column-list .job-state .item-end{
+    display: block;
+    clear: both;
+    width: 100%;
+    height: 2px;
+    animation-timing-function: ease-in-out;
+  }
+  
+  /* blue */
+  .jenkins-jobs .column-list .job-state.blue .item-name{
+    color:#2574A9;
+  }
+  .jenkins-jobs .column-list .job-state.blue .item-icon{
+    background-color:  #2574A9;
+  }
+  
+  
+  .jenkins-jobs .column-list .job-state.blue_anime .item-name{
+    color:#2574A9;
+  }
+  .jenkins-jobs .column-list .job-state.blue_anime .item-end{
+    animation: blueShadowAnimation 2s infinite;
+    background-color: #2574A9;
+  }
+  
+  .jenkins-jobs .column-list .job-state.blue_anime .item-icon{
+    background-color: transparent;
+  }
+  
+  
+  /* red */
+  .jenkins-jobs .column-list .job-state.red .item-name{
+    color:#b61a40;
+  }
+  .jenkins-jobs .column-list .job-state.red .item-icon{
+    background-color:  #b61a40;
+  }
+  
+  .jenkins-jobs .column-list .job-state.red_anime .item-name,
+  .jenkins-jobs .column-list .job-state.aborted_anime .item-name{
+    color:#b61a40;
+  }
+  .jenkins-jobs .column-list .job-state.red_anime .item-end,
+  .jenkins-jobs .column-list .job-state.aborted_anime .item-end{
+    animation: redShadowAnimation 2s infinite;
+    background-color: #b61a40;
+  }
+  
+  .jenkins-jobs .column-list .job-state.red_anime .item-icon,
+  .jenkins-jobs .column-list .job-state.aborted_anime .item-icon{
+    background-color: transparent;
+  }
+  
+  /* yellow */
+  .jenkins-jobs .column-list .job-state.yellow .item-name{
+    color:#b6b31a;
+  }
+  .jenkins-jobs .column-list .job-state.yellow .item-icon{
+    background-color:  #b6b31a;
+  }
+  
+  .jenkins-jobs .column-list .job-state.yellow_anime .item-name{
+    color:#5c5a00;
+  }
+  .jenkins-jobs .column-list .job-state.yellow_anime .item-end{
+    animation: yellowShadowAnimation 2s infinite;
+    background-color: #b6b31a;
+  }
+  
+  .jenkins-jobs .column-list .job-state.yellow_anime .item-icon{
+    background-color: transparent;
+  }
+  
+  /* grey */
+  .jenkins-jobs .column-list .job-state.grey .item-name{
+    color:grey;
+  }
+  .jenkins-jobs .column-list .job-state.grey .item-icon{
+    background-color:  grey;
+  }
+  .jenkins-jobs .column-list .job-state.grey_anime .item-name{
+    color:#464646;
+  }
+  .jenkins-jobs .column-list .job-state.grey_anime .item-end{
+    animation: greyShadowAnimation 2s infinite;
+    background-color: grey;
+  }
+  
+  .jenkins-jobs .column-list .job-state.grey_anime .item-icon{
+    background-color: transparent;
+  }
+  
+  
+  
+  @keyframes blueShadowAnimation {
+    0% {
+      -moz-box-shadow: 0px 0px 0px 0px #2574A9;
+      -webkit-box-shadow: 0px 0px 0px 0px #2574A9;
+      -o-box-shadow: 0px 0px 0px 0px #2574A9;
+      box-shadow: 0px 0px 0px 0px #2574A9;

```

```suggestion
      box-shadow: 0 0 0 0 #2574A9;
```
Et pour tout le reste aussi :D